thoth/storages/graph/postgres.py

```diff
@@ -2894,6 +2894,68 @@ def get_python_package_version_hashes_sha256(
         )
         return [item[0] for item in query.all()]
 
+    def get_python_package_required_symbols(
+        self, package_name: str, package_version: str, index_url: str
+    ) -> List[str]:
+        """Get required symbols for a Python package in a specified version."""
+        package_name = self.normalize_python_package_name(package_name)
+        package_version = self.normalize_python_package_version(package_version)
+
+        if index_url is not None:
```

Comment:
`index_url` will never be `None` based on the function signature. You can drop the `else` branch.